plc4j/spi/src/main/java/org/apache/plc4x/java/spi/generation/WriteBufferByteBased.java

```diff
@@ -392,8 +396,8 @@ public void writeString(String logicalName, int bitLength, String value, WithWri
             case "UTF16LE":
             case "UTF16BE": {
                 bytes = value.getBytes(StandardCharsets.UTF_16);
-                if(bytes.length > 2) {
-                    bytes = new byte[] {
+                if (bytes.length > 2) {
+                    bytes = new byte[]{
```

Comment:
Admittedly I don#t quite understand neither the old nor the new version of this ;-)